swiftshield-Sources/SwiftAttribute.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  SwiftAttribute
+//  swiftshield
+//
+//  Created by Weidian on 22/12/19.
+//  Copyright © 2019 Bruno Rocha. All rights reserved.
+//
+
+import Foundation
+
+enum SwiftAttribute: String {
```

Comment:
I think `SwiftAccessControl` is the proper naming here 🙂 What do you think?

Reply:
Yup, sounds better.